cookietemple/cookietemple_cli.py

```diff
@@ -94,7 +94,7 @@ def create(domain: str) -> None:
 
 
 @cookietemple_cli.command(short_help='Lint your existing cookietemple project.', cls=CustomHelpSubcommand)
-@click.argument('project_dir', type=click.Path(), default=Path(str(Path.cwd())), helpmsg='Relative path to projects directory.', cls=CustomArg)
+@click.argument('project_dir', type=click.Path(), default=Path(str(Path.cwd())), helpmsg='Relative path to projects directory.', cls=CustomArg)  # type: ignore
 @click.option('--skip-external', is_flag=True, help='Only run cookietemple linting and not external linters.')
```

Comment:
might need to refactor this here: type and default are looking weird, just default to `Path.cwd()`

Reply:
Yeah. But note that this PR does not change anything here.

Reply:
Yes, I know, I just noticed it ;D